### DISCUSSION TITLE
Environment Variable "MONGODB_URI" references Secret "next_example_mongodb_uri", which does not exist.

### DISCUSSION THREAD

**Discussion by ats1999** [vercel/next.js#17670] [Help]
While deploying `with-mongodb-mongoose`  i am getting this error on vercel `Environment Variable "MONGODB_URI" references Secret "next_example_mongodb_uri", which does not exist.` 
I am writing the environment variable in `vercel` page where they ask at the time of deployment.

**Comment by thawinwats** [comment]
You have to edit at vercel.json in the 

![Screen Shot 2563-12-05 at 3 22 09 PM](https://user-images.githubusercontent.com/18025306/101237722-a65ac080-370d-11eb-9266-567df9e455da.png)
root of your project

![Screen Shot 2563-12-05 at 3 20 58 PM](https://user-images.githubusercontent.com/18025306/101237703-7c090300-370d-11eb-8594-150d6882b5bf.png)

**Reply by geniuscd** [reply]
I'm pretty sure there is another way to add the environment variables. `vercel.json` is not included in `.gitignore` file, therefore you should avoid adding your secrets there.
@ats1999 I also added them into `vercel` environment page, but nothing happened, i kept getting this error.


So I read the documentation, and it seams `vercel.json env` variable is a absolete now. so just delete the file and you're good to go. Because in `dbConnect.js`, `process.env.MONGODB_URI` is used, the global environments in `vercel` settings page should work fine.

I think someone should remove the `verce.json` from the example repo
